examples/java/spring-boot/src/test/java/uk/gov/api/springboot/models/metadata/DataTest.java

```diff
@@ -0,0 +1,80 @@
+package uk.gov.api.springboot.models.metadata;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.*;
+
+import java.io.IOException;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.json.JsonTest;
+import org.springframework.boot.test.json.JacksonTester;
+import org.springframework.boot.test.json.JsonContent;
+
+@JsonTest
+class DataTest {
+
+  @Autowired private JacksonTester<Data> jacksonTester;
+
+  private JsonContent<Data> jsonContent;
+
+  @Nested
+  public class Serialization {
+    @BeforeEach
+    void setup() throws IOException {
+      Data data = new Data();
+      data.setName("some name");
+      data.setDescription("some description");
+      data.setUrl("some url");
+      data.setContact("some contact");
+      data.setOrganisation("some organisation");
+      data.setDocumentationUrl("some documentation url");
+      jsonContent = jacksonTester.write(data);
+    }
+
+    @Test
+    void nameIsSerialized() throws IOException {
+      assertThat(jsonContent).extractingJsonPathStringValue("name").isEqualTo("some name");
+    }
+
+    @Test
+    void descriptionIsSerialized() throws IOException {
+      assertThat(jsonContent)
+          .extractingJsonPathStringValue("description")
+          .isEqualTo("some description");
+    }
+
+    @Test
+    void urlIsSerialized() throws IOException {
+      assertThat(jsonContent).extractingJsonPathStringValue("url").isEqualTo("some url");
+    }
+
+    @Test
+    void contactIsSerialized() throws IOException {
+      assertThat(jsonContent).hasJsonPathStringValue("contact", "some contact");
+    }
+
+    @Test
+    void organisationIsSerialized() throws IOException {
+      assertThat(jsonContent).hasJsonPathStringValue("organisation", "some organisation");
+    }
+
+    @Test
+    void documentationUrlIsSerialized() throws IOException {
+      assertThat(jsonContent).hasJsonPathStringValue("documentation-url", "some documentation url");
+    }
+  }
```

Comment:
IntelliJ shows these are not required:

```suggestion
    @Test
    void nameIsSerialized() {
      assertThat(jsonContent).extractingJsonPathStringValue("name").isEqualTo("some name");
    }

    @Test
    void descriptionIsSerialized() {
      assertThat(jsonContent)
          .extractingJsonPathStringValue("description")
          .isEqualTo("some description");
    }

    @Test
    void urlIsSerialized() {
      assertThat(jsonContent).extractingJsonPathStringValue("url").isEqualTo("some url");
    }

    @Test
    void contactIsSerialized() {
      assertThat(jsonContent).hasJsonPathStringValue("contact", "some contact");
    }

    @Test
    void organisationIsSerialized() {
      assertThat(jsonContent).hasJsonPathStringValue("organisation", "some organisation");
    }

    @Test
    void documentationUrlIsSerialized() {
      assertThat(jsonContent).hasJsonPathStringValue("documentation-url", "some documentation url");
    }
  }
```